src/main/java/com/adyen/service/resource/binlookup/Get3dsAvailability.java

```diff
@@ -0,0 +1,15 @@
+package com.adyen.service.resource.binlookup;
+
+import java.util.Arrays;
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.service.Resource;
+
+public class Get3dsAvailability extends Resource {
+
+    public Get3dsAvailability(Service service) {
+        super(service,
+                service.getClient().getConfig().getEndpoint() + "/pal/servlet/BinLookup/" + Client.BIN_LOOKUP_API_VERSION + "/get3dsAvailability",
```

Comment:
we can add this as a constant in Apiconstants "/pal/servlet/BinLookup/" so that if URL changes we don't need to updated in 2 classes.

Reply:
Added constant in Client with the other urls